README.md

```
@@ -44,32 +45,29 @@ Follow these steps to set up the project locally.
     The app will run at `http://localhost:3000`.
 
 4.  **Build for production**
     ```bash
     npm run build
     ```
-    This creates an optimized build in the `build` folder.
+    This creates an optimized production build.
 
 ## 📂 Project Structure
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)